lab2/XYHVPath path
string type
---
bool success
